package.json

```diff
@@ -24,16 +24,27 @@
     "prepublish": "grunt peerDepInstall",
     "test": "grunt test"
   },
+  "peerDependencies": {
+    "@dojo/widget-core": "next",
+    "@dojo/core": "next",
+    "@dojo/has": "next",
+    "@dojo/i18n": "next",
+    "@dojo/shim": "next"
+  },
   "devDependencies": {
+    "@dojo/interfaces": "next",
+    "@dojo/loader": "next",
     "@types/chai": "4.0.*",
     "@types/glob": "5.0.*",
     "@types/grunt": "0.4.*",
     "@types/sinon": "^1.16.31",
     "codecov.io": "0.1.6",
-    "@dojo/loader": "next",
     "grunt": "~1.0.1",
     "grunt-dojo2": "latest",
     "intern": "^3.4.6",
     "typescript": "~2.4.2"
+  },
+  "dependencies": {
+    "redux": "^3.7.2"
```

Comment:
So, anyone using `interop` is going to have to pull in `redux`?  Would it make sense to just have it as a dev dependency and then the end user chooses what sort of dependencies/parts of interop they are using?

Reply:
yeah perhaps that is better.

Reply:
Thinking about this, perhaps they do have to pull in redux... because otherwise the more likely mistake will be they don't install redux at all!

After all it will only get included in the bundle if they actually use it.

Reply:
I guess I can see both sides of it... but, it means that this will grow and grow and grow in dependencies whenever it is installed as we increase the capabilities of the package.  I wasn't planning on adding `dojo` and `dijit` as dependencies when we introduce the `DijitWrapper`.  It would just seem to me that it would be "bring your own production libraries" sort of thoughts, with the code here as loosely coupled as possible.

After all, the build will error if they don't install it. 😉

Reply:
How about a peer dependency? seems like that would be the most appropriate... I think you'd actually be okay to add `dojo` and `dijit` too :)

Reply:
Gotta ❤️ `peerDependency`... only thing is _if_ I want to use `DijitWrapper` then `react` wouldn't be a dependency at all.  There is no easy way to deal with that, except for maybe [`optionalDependencies`](https://docs.npmjs.com/files/package.json#optionaldependencies)?